org.civicrm.reqangular/src/common/decorators/angular-date/datepicker-addon.js

```diff
@@ -1,35 +1,88 @@
-define([], function () {
+/* eslint-env amd */
```

Comment:
does the decorator need to be under `angular-date`? this module is really something that i would like to get rid of in the future

Reply:
Moved it outside, as discussed.